.ci/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {label "clx-host-21514||clx-host-21518"}
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 3, unit: 'MINUTES')
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        SRC_ROOT = "${env.WORKSPACE}"
+        ARTEFACT_DIR = "${env.WORKSPACE}/artefacts"
+        NFS_WORKSPACE = "/.autodirect/mtrswgwork/_ml_devops_team/workspace/ml-nccl-rdma-sharp-plugins-pr/${env.BUILD_NUMBER}"
+        CUDA_HOME = "/hpc/local/oss/cuda10.0"
+        NCCL_DIR = "/.autodirect/mtrswgwork/artemry/ci_tools_do_not_remove/nccl-sharp-stable-2019-08-16/build"
+        NCCL_TESTS_DIR = "${NFS_WORKSPACE}/nccl-tests"
```

Comment:
why need nfs dir at all ? what is there? something pre-built/pre-cached? or for mpirun?

Reply:
There're MPI app binaries (NCCL tests) which should be available across 2-node cluster.